### ISSUE TITLE
Cannot use 'quasar wrap' on Windows

### ISSUE THREAD

**Issue by kamilersz** [rstoenescu/quasar-cli#1]
I cannot run 'quasar wrap' command on Windows even though I've installed cordova globally

```
C:\Users\kamilersz\quasar\proj>quasar wrap
 i Generating Cordova wrapper for Quasar App...
{ [Error: spawn cordova ENOENT]
  code: 'ENOENT',
  errno: 'ENOENT',
  syscall: 'spawn cordova',
  path: 'cordova',
  spawnargs: [ 'create', 'wrapper' ] }
 x You need Cordova globally installed (npm install -g cordova).

C:\Users\kamilersz\quasar\proj>cordova -v
6.2.0

C:\Users\kamilersz\quasar\proj>npm -v
3.6.0

C:\Users\kamilersz\quasar\proj>node -v
v5.7.0

```

**Comment by kamilersz** [comment]
There might be clues in here as global npm package execution entry point on Windows uses .bat file
https://nodejs.org/api/child_process.html#child_process_spawning_bat_and_cmd_files_on_windows

**Comment by rstoenescu** [comment]
Setting up a VM with Windows and investigating.

**Comment by rstoenescu** [comment]
Released Quasar CLI v0.1.2 which fixes the issue. Please read [Wrapper documentation](http://quasar-framework.org/guide/cli-commands.html#Native-Mobile-App-Wrapper) if you encounter permission errors.
